src/Microsoft.DotNet.Cli.Utils/CommandResolution/WindowsExePreferredCommandSpecFactory.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.DotNet.ProjectModel;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.Extensions.PlatformAbstractions;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class WindowsExePreferredCommandSpecFactory : IPlatformCommandSpecFactory
+    {
+        public CommandSpec CreateCommandSpec(
```

Comment:
I'm going to create an issue regarding the logic in this file, and we can work forward to what it should be.

It's carrying over the logic we had previously but I'm not sure it is correct. When we close on that, I will work this file into the design more cleanly. For now it is doing what it was doing before.